Dockerfile

```diff
@@ -172,31 +172,6 @@ USER easy_infra
 
 FROM minimal AS final
 
-USER root
```

Comment:
@JonZeolla can you please explain why terrascan is being removed in the PR ?

Reply:
Yes, Packer, tfsec, and Terrascan are all being removed as they provide very little incremental benefit.  I'm moving the project to provide a single security tool per functional tool (checkov for terraform, kics for Ansible, etc.).  This is a part of